syserol/impute.py

```diff
@@ -35,3 +39,37 @@ def evaluate_missing(comps, numSample, chords):
         R2X[ii] = 1.0 - np.nanvar(tensorR - imputeVals) / np.nanvar(imputeVals)
 
     return R2X
+
+def evaluate_missing_meas(comps, numSample=15):
```

Comment:
This largely duplicates the function above. Add an option to control behavior within the existing function.